### ISSUE TITLE
RSS Feeds for Weekly are hard to reach for

### ISSUE THREAD

**Issue by zoffixznet** [perl6/user-experience#21]
There are two RSS feeds: posts/comments. They're at the very bottom of the page though, and any time you scroll down enough to make them visible, the infini-scroll loads more stuff on the page, pushing them further down.

The only way I managed to click on them is by holding down PgDn key to scroll faster than loading of more items.

**Comment by lizmat** [comment]
> On 1 May 2018, at 16:18, Zoffix Znet <notifications@github.com> wrote:
> There are two RSS feeds: posts/comments. They're at the very bottom of the page though, and any time you scroll down enough to make them visible, the infini-scroll loads more stuff on the page, pushing them further down.
> 
> The only way I managed to click on them is by holding down PgDn key to scroll faster than loading of more items.

It haz RSS feeds?

**Comment by zoffixznet** [comment]
> It haz RSS feeds?

Yes, one is for posts ( https://p6weekly.wordpress.com/feed/ ) and one is for comments ( https://p6weekly.wordpress.com/comments/feed/ )

![z](https://user-images.githubusercontent.com/5747918/39481869-e58dc2a8-4d3a-11e8-8eb2-7cc77f1aa631.png)

**Comment by timo** [comment]
I put links to the RSS feeds in the menu that you can get with the hamburger menu button on the side (or at the top for tablets or phones).

It's not much better, but at least you can click the darned thing at all!
